behat.yml.dist

```diff
@@ -0,0 +1,11 @@
+default:
+    suites:
+        default:
+            contexts:
+                - Zenstruck\Foundry\Tests\Behat\TestContext
+                - Zenstruck\Foundry\Test\Behat\FactoriesContext
+    extensions:
+        FriendsOfBehat\SymfonyExtension:
```

Comment:
I need some help figuring out why this doesn't seem to be working. When I run vendor/bin/behat I get the following error:

> Can not find a matching value for an argument `$container` of the method `Zenstruck\Foundry\Test\Behat\FactoriesContext::__construct()`.

I've confirmed the kernel isn't being booted but not sure why.

Reply:
I think the problem is to do with this: src/Listener/KernelOrchestrator.php:tearDown in symfony extension...

Reply:
Its loaded in the service extension here:
\FriendsOfBehat\SymfonyExtension\ServiceContainer\SymfonyExtension::loadKernelRebooter

Reply:
I would probably try to restructure the contexts here as a listener (like KernelOrchestrator) and perhaps try to use the priority (int) on the subscribed callbacks (e.g. 16 or something) to trigger database/foundry registry reset. Ive found it possible to induce memory leaks quite easily or race conditions in a few integrations ive attempted previously.